tests/edit_environment.rb

```diff
@@ -19,32 +19,64 @@
 
 require 'json'
 require 'uri'
+require 'open3'
 
 def get_name_server
   ns = File.readlines('/etc/resolv.conf').select do |ll|
     ll.include?('nameserver')
   end.first.chomp.gsub(/^\s*nameserver\s*/,'')
 
-  if ns == '127.0.0.1'
+  # see if we have a DHCP leases file to parse
+  if !ns or ns.start_with? '127.'
```

Comment:
Should we be using `||` rather than `or` here to prevent possible precedence issues?